http/src/request/channel/message/create_message.rs

```diff
@@ -214,20 +214,20 @@ impl<'a> CreateMessage<'a> {
     }
 
     /// Whether to fail sending if the reply no longer exists.
```

Comment:
If not set, Discord set [the value to `true` by default](https://discord.com/developers/docs/resources/channel#message-reference-object). Perhaps it should be mentioned here.

```suggestion
    /// Whether to fail sending if the reply no longer exists.
    ///
    /// Defaults to [`true`].
```